packages/ember-intl/tests/unit/macros/format-number-test.js

```diff
@@ -0,0 +1,54 @@
+import Ember from 'ember';
+import { moduleFor, test } from 'ember-qunit';
+import { formatNumber, raw, ref } from 'ember-intl/macros';
+
+moduleFor('service:intl', 'Unit | Macro | formatNumber', {
+  integration: true,
+  beforeEach() {
+    let intl = this.subject();
+    intl.setLocale('en');
+
+    this.object = Ember.Object.extend({
+      intl,
+      someNumber: 12345,
+      fractionDigits: 3,
+      formattedNumber: formatNumber('someNumber'),
+      rawFormattedNumber: formatNumber(raw(10001)),
```

Comment:
yeah, that is the one thing I'm not sure about yet. there are macros where it's a lot more common to use references over static values, at least for the positional parameters, and forcing to use `ref` there might clutter the code quite a bit. I'm 50/50 on this...